src/XMakeBuildEngine/Definition/ToolsetLocalReader.cs

```diff
@@ -57,6 +57,7 @@ protected override IEnumerable<ToolsetPropertyDefinition> ToolsVersions
         protected override IEnumerable<ToolsetPropertyDefinition> GetPropertyDefinitions(string toolsVersion)
         {
             yield return new ToolsetPropertyDefinition(MSBuildConstants.ToolsPath, BuildEnvironmentHelper.Instance.CurrentMSBuildToolsDirectory, _sourceLocation);
+            yield return new ToolsetPropertyDefinition(MSBuildConstants.SdksPath, BuildEnvironmentHelper.Instance.MSBuildSDKsPath, _sourceLocation);
```

Comment:
Not by MSBuild itself, but the first iteration of some of the magic SDKs plan to do SDK-to-SDK references using the property. It's set here for .NET Core and in app.config for full.